NDTensors/test/ITensors/TestITensorDMRG/TestITensorDMRG.jl

```diff
@@ -0,0 +1,17 @@
+## This is getting closer still working on it. No need to review
+## Failing for CUDA mostly with eigen (I believe there is some noise in
+## eigen decomp with CUBLAS to give slightly different answer than BLAS)
+module TestITensorDMRG
+using ITensors
+using NDTensors
+
+reference_energies = Dict([
+  (4, -1.6160254037844384), (8, -3.374932598687889), (10, -4.258035207282885)
+])
+
+function get_ref_value(device, sites, elt)
```

Comment:
Seems like this doesn't need `device` or `elt` anymore.

Even simpler would be to just directly access `reference_energies` and not have this function.